manifests/m/MicroSIP/MicroSIP/3.20.7/MicroSIP.MicroSIP.installer.yaml

```diff
@@ -0,0 +1,25 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: MicroSIP.MicroSIP
+PackageVersion: 3.20.7
+MinimumOSVersion: 10.0.0.0
+Protocols:
+- sip
+- tel
+InstallModes:
+- interactive
+- silent
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: nullsoft
+  Scope: machine
+  InstallerUrl: https://www.microsip.org/download/MicroSIP-3.20.7.exe
+  InstallerSha256: 30CFDC2A9BF193931A4B168104440A31F61CFB13869085E3CDFEBF3F908FBAD4
+  InstallerSwitches:
+    Silent: /S
```

Comment:
```suggestion
```
I believe these aren't necessary as the client automatically adds the switches based on `InstallerType`.